EntAppProject/src/main/java/com/jobook/EntAppProject/Services/UserServiceStub2.java

```diff
@@ -0,0 +1,19 @@
+package com.jobook.EntAppProject.Services;
+
+import org.springframework.stereotype.Component;
+
+import com.jobook.EntAppProject.dto.UserDTO;
+
+@Component
+public class UserServiceStub2 {
+
+	public UserDTO fetchById(int id) {
+		UserDTO userDTO = new UserDTO();
+		userDTO.setId(12);
+		userDTO.setUserName("BearcatUC");
+		return userDTO;
+	}
+	
+	public void save(UserDTO userDTO) {
+	}
+}
```

Comment:
Created a DTO similar to the one shown in class

Reply:
A couple of things:
1) This isn't implementing an interface.  A stub indicates that there will be (or is) a working version somewhere... and these should be linked by an interface, so that they can be swaped out independently.
2) As mentioned above... change the name.  Adding a number to the end of a class name often indicates technical debt.  Why not name it UserServiceStub?  If one already exists, consider refactoring and moving it here.